Maskbook/Extension/Optional+Extension.swift

```diff
@@ -30,3 +30,21 @@ extension Optional where Wrapped == Bool {
         }
     }
 }
+
+extension Optional {
+    var isSome: Bool {
+        switch self {
+        case .none: return false
+        case .some: return true
```

Comment:
suggestion `return self == .none` return self == .some`

Reply:
The `Wrapped` will need to conform to `Equatable`, this is not we want.